src/frontend/src/eth/components/send/SendTokenModal.svelte

```diff
@@ -67,6 +67,11 @@
 
 			currentStep = undefined;
 		});
+
+	const goToWizardStep = (stepName: WizardStepsSend) => {
+		const stepNumber = steps.findIndex(({ name }) => name === stepName);
```

Comment:
What happens if resolved step is -1 ?

Reply:
wow! I thought I had corrected it! sorry, changing it